### DISCUSSION TITLE
Need for peak re-calling after Cell Ranger arc v2?

### DISCUSSION THREAD

**Discussion by AmelZulji** [stuart-lab/signac#1674] [Q&A]
Hi there, I was wondering whether peak re-calling using MACS2 as described [here](https://stuartlab.org/signac/articles/pbmc_multiomic#peak-calling) is still beneficial given that CellRanger arc v2 has improved the algorithm as described [here](https://www.10xgenomics.com/blog/ask-the-10x-perts-4-reasons-to-upgrade-your-cell-ranger-atac-and-arc-software)?

Thank you for help!

**Comment by ShanshanHe-44** [comment]
That is an interesting discussion. I am also curious about the performance of the updated CellRanger v2 compared with MACS2. The current peak calling vignette [here](https://stuartlab.org/signac/articles/peak_calling) is still using the PBMCs 10k dataset based on CellRanger pipeline version 1.1.0. From the coverage plot, the MACS2 peaks appear to be more precise. However, I am also interested in how CellRanger v2 performs. 
As far as I know, CellRanger v2 doesn't include the summit position of each peak, whereas MACS2 does. So, if your research interest is understanding motifs, MACS2 might still be the better choice.
![coverage plot](https://stuartlab.org/signac/articles/peak_calling_files/figure-html/unnamed-chunk-4-1.png)
